api/CustomElementRegistry.json

```diff
@@ -335,10 +335,14 @@
               "version_added": "10.3",
               "notes": "Supports 'Autonomous custom elements' but not 'Customized built-in elements'"
             },
-            "samsunginternet_android": {
-              "notes": "Support for 'Autonomous custom elements' only.",
-              "version_added": "6.0"
-            },
+            "samsunginternet_android": [
+              {
+                "version_added": "9.0"
```

Comment:
This needs partial reversion.

It seems someone has kindly updated this and we don't want to overwrite their work.

Reply:
Port comments back.